applications/SolversApplication/custom_solvers/solution_schemes/eigensolver_scheme.hpp

```diff
@@ -1,12 +1,12 @@
 //
-//   Project Name:        KratosSolidMechanicsApplication $
-//   Created by:          $Author:   michael.andre@tum.de $
-//   Last modified by:    $Co-Author:                     $
-//   Date:                $Date:           September 2016 $
-//   Revision:            $Revision:                  0.0 $
+//   Project Name:        KratosSolversApplication $
+//   Created by:          $Author:     JMCarbonell $
```

Comment:
Can we focus on this PR? If things were incorrectly done in the past (arguable), that is no reason to keep doing them wrong.

Reply:
It is the one from StructuralMechanics, which was migrated to SolidMechanics in #736

The original files were added here: https://github.com/KratosMultiphysics/Kratos/commit/39193d50de7053516f39f6c1e75da9976fc4ea6b#diff-ae31bd10d286705c1ab8892043f27b7a
(almost one year before it was copied to Solid...)

Reply:
I took this as reference:
mechanical_explicit_strategy.hpp
//  Main authors:    Klaus B Sautter (based on the work of JMCarbonel)
explicit_central_differences_scheme.hpp:
//  Main authors:    Klaus B Sautter (based on the work of MSantasusana)

Reply:
There it was properly referenced where the files came from

Also these files were heavily modified to account for the rotations (e.g. of shell & beam-elements)

Reply:
The reference it was exactly the same. It sounds like, when I do myself is ok, when others do it is wrong
I have updated the headers to avoid these conflicts. 

P.D. There is a typo in (based on the work of JMCarbonel), my surname is Carbonell.